target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -216,6 +220,9 @@ ramips_setup_macs()
 	zbtlink,zbt-we1226)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x2e)" 1)
 		;;
+	zyxel,keenetic-4g)
+		wan_mac=$(mtd_get_mac_binary factory 0x28)
```

Comment:
What's the MAC assignment on vendor firmware? Which addresses are used for LAN/WAN/Wifi?

Which addresses are found on flash in locations 0x4, 0x28, 0x2e, 0x8004, 0xe000, 0xe006?